app/themes/user-admin/styles/base/_styles.scss

```diff
@@ -46,6 +46,27 @@ h1, h2, h3, h4 {
 }
 
 
+/* Square corners
+-------------------------------------------------- */
+.square-corners, .alert,
```

Comment:
We need to target all elements with the rounded corners, not create a new CSS class.

See my previous note for some clarification.